.github/workflows/ci.yml

```diff
@@ -108,13 +108,19 @@ jobs:
       # Enable corepack for packageManager config
       - run: corepack enable || sudo corepack enable
       - run: yarn install
-      # Set up docker-bake files used by docker/bake-action
-      - run: node_modules/.bin/synthetic-chain prepare-ci
+
+      - run: docker system df
+      - run: docker buildx du --verbose
+      - run: df -h
```

Comment:
these blocks aren't necessary for main, but I'd like to leave it in this PR.

1. to not delay merge by another run (taking over an hour)
2. to have the diagnostics a little longer while the change settles

If #2 is not compelling, I'm open to cutting e8fce151e00f959fe1b975602c8669dfdc7318ec from this branch and merging without another green.

Reply:
diagnostics of this sort are somewhere between cost-effective and priceless, IME. (unless they're very, very verbose; I haven't checked in this case).